www/i18n/en-US.js

```diff
@@ -348,6 +348,9 @@ export default {
   "automaticallyProceedToNextGameAfterMoving": "Automatically proceed to next game after moving",
   "autoSwitch": "Auto switch",
   "puzzles": "Puzzles",
+  "coordinates": "Coordinates",
+  "coordinates-training": "Coordinates Training",
+  "startTraining": "Start Training",
```

Comment:
Our translations are sourced from [CrowdIn](https://crowdin.com/translate/lichess), so just adding these translations in this locale file won't do anything. Fortunately, these keys are already translated :)

You'll need to add the `coordinates` module here: https://github.com/veloce/lichobile/blob/b7c3a0b8f1d304f3ee5cdb639c6a6bee82ea74be/scripts/i18n-update.ts#L12

and then re-run the script, but then you'll get these translations for free. (Note that the keys may be different.)

Reply:
👍

Reply:
Thanks, fixed!
But could you elaborate more how does this work? How by adding the module name to that array I got the correct translation?

Reply:
Sure, the code in `scripts/i18n-update.ts` basically does the following:
1. Download and unzip the complete `lila` translation package, which is organized into "modules" (which are just separate translation files + folders)
2. Copy translations from only the modules (folders) that we use, for performance/optimization purposes

So adding the `coordinates` string to that array indicated that the script should also import the translations from the `coordinates` module/folder. That's all :)

Reply:
I see, this is lila translation which is used by lichess web as well, also `coordinates` is the name they use in this package not the name of the module I wrote.